src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -438,15 +438,16 @@ private void processAbort(
             int index,
             int length)
     {
-        factory.correlations.remove(acceptCorrelationId);
+        factory.writer.doAbort(acceptReply, replyStreamId);
         if (targetBeginIssued)
         {
             factory.writer.doAbort(target, targetId);
-            cleanupConnectReply.run();
+
         }
-        else
+        if (factory.correlations.remove(acceptCorrelationId) == null &&
```

Comment:
Given the important side effect of this line, please move the result to a variable called `correlation`, then use `correlation == null` as part of the `if `condition, rather than embedding the call to `remove` in the `if` condition directly.